crates/ark/src/help_proxy.rs

```diff
@@ -17,6 +17,10 @@ use actix_web::HttpServer;
 use harp::exec::RFunction;
 use harp::exec::RFunctionExt;
 use mime_guess::from_path;
+use reqwest::Client;
+use reqwest_middleware::ClientBuilder;
+use reqwest_retry::policies::ExponentialBackoff;
+use reqwest_retry::RetryTransientMiddleware;
```

Comment:
The reqwest crate doesn't have any retry mechanism, but we can use these related crates to set up middleware and retries.